docs/topics/gradle/gradle-configure-project.md

```diff
@@ -386,6 +386,64 @@ integrationTestCompilation {
 Here, the `integrationTest` compilation is associated with the `main` compilation that gives access to `internal`
 objects from functional tests.
 
+### Configure with Java Modules (JPMS) enabled
```

Comment:
Does it make sense to add that JDK 9 at least is required to work with modules? 
And specify, how to set toolchain version explicitly.

Reply:
>Does it make sense to add that JDK 9 at least is required to work with modules?

I don't think so cause it's not our requirement. Those who use modules already use them. Or if they learn about them from our documentation, they will probably go elsewhere to obtain more knowledge.